### PR TITLE
Update troubleshooting section

### PR DESCRIPTION
**Pull request by MarceloSalazar**
@AnotherButler please have a look. If you're happy with this, I'll update all examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,16 +72,4 @@ To debug the application:
...
-
- If not, you can update it:
-
- ```
- pip install mbed-cli --upgrade
- ```
-
-2. If using Keil MDK, make sure you have a license installed. [MDK-Lite](http://www.keil.com/arm/mdk.asp) has a 32 KB restriction on code size.
\ No newline at end of file
+If you have problems, you can review the [documentation](https://os.mbed.com/docs/latest/tutorials/debugging.html) for suggestions on what could be wrong.
```
Comment (AnotherButler) [inline README.md]: Query: Should we add "and how to fix it" to the end?

Comment (MarceloSalazar) [reply README.md]: Yes. Feel free to tweak the doc and then merge. Thanks!

**Comment by AnotherButler** [comment]
@MarceloSalazar I don't have permissions, so I created another PR.

**Comment by MarceloSalazar** [comment]
Merging https://github.com/ARMmbed/mbed-os-example-blinky/pull/106

**Pull request merged**